README.md

```diff
@@ -101,6 +101,12 @@ npm run test
 
 For more infos about how tests are written, see [here](https://github.com/vpdb/vpx-js/tree/master/test#readme)
 
+## VPX Fileformat
+
+VPX files using Microsoft OLE2 files (also called Structured Storage, Compound File Binary Format or Compound Document File Format). Some links to work (extract/repack) those files:
+- [oletools]: https://www.decalage.info/python/oletools
+- [Visual Pinball]: https://vpinball.com/

```

Comment:
I don't think this adds any value to the main readme. And VP's official URL is https://sourceforge.net/projects/vpinball/

Reply:
VPX files is the main file for vpx-js, so not sure if I agree. But I can remove this if you don't agree